src/ploomber/util/util.py

```diff
@@ -82,6 +82,20 @@ def _make_requires_error_message(missing_pkgs, fn_name, extra_msg):
     return error_msg
 
 
+def check_mixed_envs():
+    try:
+        from pip._internal.operations import freeze
+    except ImportError:
+        from pip.operations import freeze
```

Comment:
let's make this function take the output of `pip freeze` as input (the string output). because when running `ploomber install` the user may create a new environment, hence we want to check the freeze in that env, not in the current one.

```python
def check_mixed_envs(text):
    ...
```